src/nupic/engine/Input.cpp

```diff
@@ -53,9 +53,9 @@ Input::~Input()
   }
 }
 
+
 void
-Input::addLink(const std::string& linkType, const std::string& linkParams,
-               Output* srcOutput, const size_t propagationDelay)
```

Comment:
Replaced by orthogonal Link constructor plus the new addLink below that adds a Link instance to the Input/Output